db-helpers.js

```diff
@@ -103,24 +105,11 @@ function getUsers (callback) {
     })
 }
 
-function getUserByEmail (email, callback) {
+function getFormSubmission (email, dateSubmit, callback) {
     openDbConnection( (err, client, done) => {
-        client.query('SELECT * FROM users WHERE email = $1', [email], function (err, result) {
```

Comment:
Find by email is no longer a good way to find a single row, so I deleted the method. GitHub's diff makes this change look more complex than it really is. :/